ruma-events-macros/src/event.rs

```diff
@@ -77,7 +77,7 @@ pub fn expand_event(input: DeriveInput) -> syn::Result<TokenStream> {
 
                 let event_type = ::ruma_events::EventContent::event_type(&self.content);
 
-                let mut state = serializer.serialize_struct(stringify!(#ident), 7)?;
+                let mut state = serializer.serialize_struct(stringify!(#ident), 9)?;
```

Comment:
I can remove this I just wanted to remember to ask if we should be dealing with this and is there any chance this could be part of the bincode deserialization problem @MTRNord was asking about on matrix (either in the Ruma room or in matrix-rust-sdk)?

Reply:
@jplatte had some ideas why bincode doesn't work I think. json does work fine (obviously. this is matrix. if it doesn't do json this wouldn't be matrix)

Reply:
No, this has nothing to do with the bincode "expected valid JSON" error. If you want more details about that, ping me on Matrix.